### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by Ramilito** [Ramilito/kubectl.nvim#282] [General]
What would you think is missing for us to call it v1?
@mosheavni 

- [ ] Update docs #247

**Comment by mosheavni** [comment]
> What would you think is missing for us to call it v1? @mosheavni

Yes v1 sounds good! 🙌🏽

**Reply by mosheavni** [reply]
we're missing some docs though

**Reply by mosheavni** [reply]
this is harder than writing the code 😆

**Reply by Ramilito** [reply]
lol agreed!
